core/chains/evm/config/v2/defaults/Arbitrum_Goerli.toml

```diff
@@ -10,7 +10,7 @@ LimitMax = 1_000_000_000
 # Arbitrum uses the suggested gas price, so we don't want to place any limits on the minimum
 PriceMin = '0'
 PriceDefault = '0.1 gwei'
-PriceMax = '1000 gwei'
+PriceMax = '115792089237316195423570985008687907853269984665.640564039457584007913129639935 tether'
```

Comment:
Note: should probably be a nicer sentinel value for this like `max`, `unlimited`, `MaxUint256` etc